### ISSUE TITLE
wscat emits colours even when not emitting data to a terminal

### ISSUE THREAD

**Issue by dvarrazzo** [websockets/wscat#24]
Colours should be used only in a terminal, and there should be an option to disable them too.

```
$ wscat --connect  ws://somewhere | xxd | head
0000000: 1b5b 324b 1b5b 451b 5b33 326d 636f 6e6e  .[2K.[E.[32mconn
0000010: 6563 7465 6420 2870 7265 7373 2043 5452  ected (press CTR
0000020: 4c2b 4320 746f 2071 7569 7429 1b5b 3339  L+C to quit).[39
0000030: 6d0a 3e20 1b5b 324b 1b5b 451b 5b33 346d  m.> .[2K.[E.[34m
0000040: 3c20 7b22 7473 223a 3134 3738 3038 3537  < {"ts":14780857
0000050: 3139 2e38 392c 2264 6174 6122 3a5b 5b22  19.89,"data":[["
...
```

Please use [isatty](http://man7.org/linux/man-pages/man3/isatty.3.html).

**Comment by lpinca** [comment]
Switching to https://github.com/chalk/chalk should help with this as chalk automatically detects if colors can be used by checking among other things if `process.stdout` is a tty.

I'm not sure what to do about other decorators though like `<` and `>` characters.

**Comment by nwwells** [comment]
I think the same thing for this as we get for #2 -- a "raw" mode, where colors and extra characters (including state messages aren't added would be great

**Comment by nwwells** [comment]
Fixed in a2ae4f59773d5848b8a15c6dc7014ca8f8180b8c and 2de4e0f0a7341b83024cde0a3a1e7403a132f001. Will be released with 1.2.0 (see #35)
